frontend/src/pages/org/dashboard.ts

```diff
@@ -0,0 +1,273 @@
+                    })
+                  );
+                }}
+              >
+                <sl-button
+                  slot="trigger"
+                  size="small"
+                  caret
+                  ?disabled=${metrics.storageQuotaBytes > 0 &&
+                  metrics.storageUsedBytes >= metrics.storageQuotaBytes}
+                >
+                  <sl-icon slot="prefix" name="plus-lg"></sl-icon>
+                  ${msg("Add New...")}
+                </sl-button>
+                <sl-menu>
+                  <sl-menu-item value="browser-profile">
+                    ${msg("Browser Profile")}
+                  </sl-menu-item>
+                  <sl-menu-item value="upload">${msg("Upload")}</sl-menu-item>
+                </sl-menu>
+              </sl-dropdown>
+            </footer> `
+          )}
+          ${this.renderCard(
+            msg("Crawling"),
+            (metrics) => html`
+              <dl>
+                ${this.renderStat({
+                  value: metrics.workflowsRunningCount,
+                  singleLabel: msg("Crawl Running"),
+                  pluralLabel: msg("Crawls Running"),
+                  iconProps: { name: "dot", library: "app" },
+                })}
+                ${this.renderStat({
+                  value: metrics.workflowsQueuedCount,
+                  singleLabel: msg("Crawl Workflow Waiting"),
+                  pluralLabel: msg("Crawl Workflows Waiting"),
+                  iconProps: { name: "hourglass-split" },
+                })}
+                ${this.renderStat({
+                  value: metrics.pageCount,
+                  singleLabel: msg("Page Crawled"),
+                  pluralLabel: msg("Pages Crawled"),
+                  iconProps: { name: "file-richtext-fill" },
+                })}
+              </dl>
+            `,
+            (metrics) => html`
+              <footer class="mt-4 flex justify-end">
+                <sl-button
+                  href=${`/orgs/${this.orgId}/workflows?new&jobType=`}
+                  size="small"
+                  @click=${this.navLink}
+                >
+                  <sl-icon slot="prefix" name="plus-lg"></sl-icon>${msg(
+                    "New Workflow"
+                  )}
+                </sl-button>
+              </footer>
+            `
+          )}
+          ${this.renderCard(
+            msg("Collections"),
+            (metrics) => html`
+              <dl>
+                ${this.renderStat({
+                  value: metrics.collectionsCount,
+                  singleLabel: msg("Collection Total"),
+                  pluralLabel: msg("Collections Total"),
+                  iconProps: { name: "collection-fill" },
+                })}
+                ${this.renderStat({
+                  value: metrics.publicCollectionsCount,
+                  singleLabel: msg("Public Collection"),
+                  pluralLabel: msg("Public Collections"),
```

Comment:
Added as a suggestion, good catch @tw4l!
```suggestion
                  singleLabel: msg("Sharable Collection"),
                  pluralLabel: msg("Sharable Collections"),
```

Reply:
```suggestion
                  singleLabel: msg("Shareable Collection"),
                  pluralLabel: msg("Shareable Collections"),
```
To match how we spell it elsewhere :)